lib/stripped-build-plugins.js

```diff
@@ -36,7 +37,26 @@ module.exports = function(environment) {
           'warn',
           'debugSeal'
         ]
-      })
+      }),
+      // comment out when running non-baseline production benchmarks
+      // WARNING do not ever commit the commented out version!
+      stripHeimdall
```

Comment:
production builds are issued on CI requests and tags auto publish to bower. I wouldn't want heimdall to get in there accidentally; historically build related changes have been difficult to track down and reproduce. something like `process.env.ENABLE_HEIMDALL` would prevent this change from accidentally being checked into source control.